xclim/core/checks.py

```diff
@@ -383,3 +316,16 @@ def missing_pct(da, freq, tolerance, **indexer):
       A boolean array set to True if period has missing values.
     """
     return MissingPct(da, freq, **indexer)(tolerance=tolerance)
+
+
+def missing_default(da, freq, **indexer):
```

Comment:
Maybe rename to "missing_from_context" ?

Reply:
Also, is there a way to avoid special casing the arguments ?

Reply:
You mean for the options like in "wmo" and "pct"? I didn't find any that fitted the `set_options` paradigm...

Reply:
There could be a dict-like option "missing_options"  that takes dict-like arguments and update itself with them?
Ex: `xc.set_options(check_missing='pct', missing_options={'pct':{'tolerance': 0.04}})`
It's a lot of nesting, but it's more generic. 
Accordingly, the `register_missing_method` would take an `options` arg to register the available options and default values.

Reply:
I was thinking about the last option, and in fact it can inspect the function automatically.